ui/widgets/nuxeo-checkmark.js

```diff
@@ -111,11 +112,21 @@ import '../nuxeo-icons.js';
       };
     }
 
+    ready() {
+      super.ready();
+      this.setAttribute('role', 'checkbox');
+      this.setAttribute('aria-checked', false);
+    }
+
     _tap() {
       if (!this.disabled) {
         this.checked = !this.checked;
       }
     }
+
+    _ariaChecked() {
+      this.setAttribute('aria-checked', this.checked);
```

Comment:
Since this is a boolean, when checked is false, we actually want to remove the attribute.

Reply:
aria-checked="false" is actually a string that needs to be present so AT can pick up its status.

Reply:
I thought it was a boolean. If it is a string, then this is just fine!